### DISCUSSION TITLE
Access to /etc and /var actual directory paths

### DISCUSSION THREAD

**Discussion by Algoss** [aquasecurity/kube-bench#1252] [Questions and Help]
While running kube-bench in kubernetes cluster inside a pod, it will need access to the host's PID namespace in order to check the running processes, as well as access to some directories on the host where config files and other files are stored.

For the PSP, can we get a list of all the Paths that kube-bench actually needs to run the tests, not the top level /etc/kubernetes and others? It is mostly important for /etc/kubernetes since there is secrets that could be stored under that subdir.

Please provide the feedback

**Comment by mozillazg** [comment]
You can turn on the verbose log output by setting the `-v 3` command line option to see all needed paths when running the tests.
